admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
+===================
+
+ownCloud supports six user types.
+These are:
+
+- `Anonymous`_
+- `Guest`_
+- `Standard User`_
+- `Federated User`_
+- `ownCloud Group Administrator`_
+- `ownCloud Administrator`_
+
```

Comment:
"Auditor" role missing on purpose?

Reply:
"System Administrator" is a very important role. Missing here.

Reply:
These were left out, as I thought the documentation was purely after software roles, not ancillary human roles. I'll add them in.

Reply:
I thought to describe how we design ownCloud from a roles standpoint to provide a better understanding how roles are supposed to be taken and why things work as they do. In the meantime I'm a little unsure if the docs really are the place to put such information as you also had a different understanding of this. I would like to leave it up to you to put these information in a documentation frame or not. But I think if we're talking about roles here we should cover all of them as suggested.